layouts/register/types/by-mail.html

```diff
@@ -1,5 +1,15 @@
 {{ $translation := ( index $.Site.Data.translations .Site.Language.Lang ) }}
 {{ $by_mail_intro := replace $translation.register.by_mail__intro "%link%" (partial "links/by-mail-link.html" .) }}
+
+{{ $onlineDeadline := cond (ne .Params.online_deadline nil) .Params.online_deadline .Params.default_online_deadline }}
+{{ $postmarkedDeadline := cond (ne .Params.mail_postmarked_deadline nil) .Params.mail_postmarked_deadline .Params.default_mail_postmarked_deadline }}
+{{ $ipDeadline := cond (ne .Params.ip_deadline nil) .Params.ip_deadline .Params.default_ip_deadline }}
+
+{{ $dates__electionday := $translation.register.dates__electionday }}
+{{ if and (eq .Params.state_name "Louisiana") (ne $translation.register.dates__electionday_la nil) }}
+{{ $dates__electionday = $translation.register.dates__electionday_la }}
+{{ end }}
```

Comment:
Once you have added the override to the state-data.json file you can recompile the content files and then you can use the same method as the deadline to simplify this logic into a single line.